neuror/main.py

```diff
@@ -238,12 +238,14 @@ def _max_y_dendritic_cylindrical_extent(neuron):
 class Repair(object):
     '''The repair class'''
 
+    # pylint: disable=dangerous-default-value
```

Comment:
This is fine, I don't mutate the dict returned by default.

Reply:
Please elaborate on that as you use compare to `None` later:
```
if repair_flags is not None:
```

Reply:
I have kept the same logic but changed the implementation.
There is no longer a dangerous-default-value